services/read.go

```diff
@@ -0,0 +1,67 @@
+package services
+
+import (
+	"fmt"
+	db "hawqal/db"
+	dbConn "hawqal/db_migrator"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+

```

Comment:
should create one local structure to make db.Conn() and should be used by all the get functions!